.circleci/config.yml

```diff
@@ -138,6 +138,27 @@ jobs:
             cd packages/*/"@sfx/${package}"
             yarn test
 
+  ### unit_test_interaction: unit_test_interaction_{package}
+  ### Runs the interaction tests for {package}.
+  unit_test_interaction:
```

Comment:
I don't think interaction tests, at least the way we've written them, count as unit tests. I'd call this `interaction_test`, then change `unit_test_final` to `test_final` or something like that.

Reply:
Updated.

Reply:
Updated.